lib/user.rb

```diff
@@ -0,0 +1,35 @@
+class User
+  attr_reader :name, :jokes
+
+  def initialize(name)
+    @name = name
+    @jokes = []
+  end
+
+  def learn(joke)
+    @jokes << joke
+  end
+
+  def tell(user, joke)
+    user.learn(joke)
```

Comment:
NICE!! Great work seeing that the first argument was really a user, so you could use the `learn` method you just wrote, instead of needing to do the shoveling work again.